py-scripts/toolbelt/client/docker.py

```diff
@@ -0,0 +1,17 @@
+from toolbelt.client.session import BaseUrlSession
+
+DOCKER_HUB_URL = "https://hub.docker.com"
+
+
+class DockerClient:
+    def __init__(self, namespace: str) -> None:
+        self._namespace = namespace
+        self._session = BaseUrlSession(DOCKER_HUB_URL)
+
+    def check_image_exists(self, repo: str, tag: str) -> bool:
```

Comment:
```suggestion
    def check_image_exists(self, repo: str, tag: str) -> Any:
```
`import Any from typing` required